BraintreeUIKit/Components/BTUIKExpiryInputView.m

```diff
@@ -98,11 +98,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
                                                                      metrics:nil
                                                                        views:viewBindings]];
         
-        id bottomReferenceView = self;
-
-        if (@available(iOS 11.0, *)) {
-            bottomReferenceView = self.safeAreaLayoutGuide;
-        }
+        UILayoutGuide *bottomReferenceView = self.safeAreaLayoutGuide;
```

Comment:
We could just remove this variable and use `self.safeAreaLayoutGuide` directly. In my mind, that helps readability since I don't have to remember what the variable refers to.

If we want to keep the variable, how about renaming it to `bottomLayoutGuide` (since it's never actually a view anymore).